python/lsst/pipe/tasks/postprocess.py

```diff
@@ -950,8 +966,8 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         inputs = butlerQC.get(inputRefs)
         self.log.info("Concatenating %s per-patch Object Tables",
                       len(inputs["inputCatalogs"]))
-        df = pd.concat(inputs["inputCatalogs"])
-        butlerQC.put(pipeBase.Struct(outputCatalog=df), outputRefs)
+        table = astropy.table.vstack(inputs["inputCatalogs"], join_type="exact")
```

Comment:
This was a major memory hog on real RC2 data before.  I assume this is also a major memory hog; I want to confirm that it isn't *more* of a memory hog now?